### ISSUE TITLE
Unlock node width and height

### ISSUE THREAD

**Issue by Niveda-S** [cytoscape/py2cytoscape#82]
Hi, 

Is there a way to unlock node width and height using py2cytoscape?
I use cytoscape version 3.7.0

Thanks!

**Comment by zachary822** [comment]
It is a little counter-intuitive but

```python
from py2cytoscape import cyrest

csc = cyrest.cyclient()

csc.styles.updateDependencies('ConnecTF', [{'visualPropertyDependency': 'nodeSizeLocked', 'enabled': True}])
```

However, before you go try this. Pull request #84 needs to be merged before any of this works. The way PUT requests are handled in the current version throws an error.

**Comment by zachary822** [comment]
It is a little counter-intuitive but

```python
from py2cytoscape import cyrest

csc = cyrest.cyclient()

csc.styles.updateDependencies('styleName', [{'visualPropertyDependency': 'nodeSizeLocked', 'enabled': False}])  # True/False turns it on or off
```

However, before you go try this. Pull request #84 needs to be merged before any of this works. The way PUT requests are handled in the current version throws an error.

**Comment by jorgeboucas** [comment]
closes with dd34de8
